src/components/TodoContainer.js

```diff
@@ -80,7 +85,7 @@ const TodoContainer = ({ tableName }) => {
           <TodoList
             todoList={todoList}
             onRemoveTodo={removeTodo}
-            isChecked={isChecked}
+            areChecked={todoList.map(({id, isChecked}) => isChecked ? id : undefined)}
```

Comment:
More notes for future 1:1 discussion: 

Here's another hint that we might ultimately want another data structure to keep track of these checked states - We'e using this `array` more like it were an `object` or a `set`

Reply:
Great job getting it working how you did btw! This is just a good window into some future topics that might make this type of thing easier going forward